compliance_suite/test_runner.py

```diff
@@ -1,8 +1,17 @@
+from unittest import skip
 from compliance_suite.tests import initiate_tests
 from compliance_suite.utils import data
 import datetime
 import re
 import sys
+import ga4gh
+#from compliance_suite.info_algorithms import *
+#from compliance_suite.metadata_algorithms import *
```

Comment:
remove the following:
```
#from compliance_suite.info_algorithms import *
#from compliance_suite.metadata_algorithms import *
```

since they are commented out